lib/ap/locale/dcngettext.c

```diff
@@ -55,7 +57,7 @@ char *bindtextdomain(const char *domainname, const char *dirname)
 		return 0;
 	}
 
-	LOCK(lock);
+	//LOCK(lock);
```

Comment:
it's much better if you do something like
if (0) LOCK(lock);
That way it will always compile.